src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html

```diff
@@ -0,0 +1,430 @@
+
+      .non-seat-group>div {
+        background-color: var(--paper-grey-500);
+        display: inline-block;
+        text-align: center;
+      }
+
+      .non-seat-group>div>div {
+        position: relative;
+        color: #fff;
+        width: 100%;
+        top: calc(50% - 0.75em);
+      }
+
+      .non-seat-group iron-icon {
+        --iron-icon-height: 56px;
+        --iron-icon-width: 56px;
+      }
+
+      .lounge {
+        width: 300px;
+        height: 450px;
+      }
+
+      .stage {
+        width: 315px;
+        height: 180px;
+        margin-top: 270px;
+      }
+
+      .entrance {
+        width: 195px;
+        height: 330px;
+        margin-top: 120px;
+      }
+
+      #listbox {
+        overflow-y: auto;
+        max-height: 40vh;
+      }
+
+      .seats {
+        display: flex;
+      }
+
+      .team-member {
+        cursor: pointer;
+      }
+    </style>
+
+    <lancie-ajax auto-fire id="ajaxGetAllSeats" refurl="seats" on-lancie-ajax="onGetAllSeats"></lancie-ajax>
+    <lancie-ajax auto-fire id="ajaxGetTeamTickets" refurl="tickets/teammembers" on-lancie-ajax="onGetTeamTickets"></lancie-ajax>
+    <lancie-ajax id="ajaxReserveSeat" method="POST" on-lancie-ajax="onReserveSeat"></lancie-ajax>
+    <lancie-ajax id="ajaxAddSeatRow" method="POST" body="[[getSeatGroupDTOBody(seatGroup,seatAmount)]]" on-lancie-ajax="onAddSeatRow" refurl = "seats"></lancie-ajax>
+    <lancie-ajax id="ajaxRemoveSeats" method="DELETE" body="[[getSeatGroupDTOBody(removeGroup,removeAmount)]]" on-lancie-ajax="onRemoveSeats" refurl = "seats"></lancie-ajax>
+    <lancie-ajax id="ajaxLockSeat" method="POST" body="[[lockBoolean]]" on-lancie-ajax="onLockSeat"></lancie-ajax>
+    <lancie-ajax id="ajaxLockGroup" method="POST" body="[[lockBoolean]]" on-lancie-ajax="onLockGroup"></lancie-ajax>
+
+    <paper-input label="Search user" value="{{filterUser}}"></paper-input>
+    <div class="seatmapwrapper">
+      <div class="seatmap">
+        <div class="seats">
+          <template is="dom-repeat" items='[[seatGroups]]'>
+            <div class="seatmap-row">
+              <div class="grouplabel" on-tap="openLockGroupDialog" data-item$="[[item]]">[[item]]</div>
+              <div class="seats-group">
+                <template is="dom-repeat" items="[[_getSeatGroup(seats, item)]]">
+                  <lancie-admin-seat tickets="[[tickets]]" filter="[[filterUser]]" seat="[[item]]" user="[[user]]" fixed="[[fixed]]"></lancie-admin-seat>
+                </template>
+              </div>
+            </div>
+          </template>
+          <div class="seatmap-row">
+            <div class="add-seat" on-tap="openAddSeatRowDialog">+</div>
+            <div class="add-seat" style="margin-top: 10px;" on-tap="openSeatRemoveDialog">-</div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <lancie-dialog id="seatDialog">
+      <h2>Seat reservation</h2>
+      <div class="dialog-content">
+        <lancie-error id="error"></lancie-error>
```

Comment:
`seatReservationError` could be a good name. Maybe a good idea to find a good format to handle all these duplicate elements with similar names, this only results in unclarity.

Reply:
Resevation was not implemented on this page. I have removed reservation functions, and renamed the error to `editSeatError`